test/functional/postsV2.js

```diff
@@ -164,6 +165,35 @@ describe('TimelinesControllerV2', () => {
           expect(post.posts.id, 'to be', lunaPost.id);
         });
       });
+
+      describe('Open Graph test', () => {
+        describe('Luna is a public user', () => {
+          it('should return information for a public post', async () => {
+            const response = await fetchPostOpenGraph(lunaPost.id);
+            response.should.include('Luna post');
+            response.should.include('og:title');
+            response.should.include('og:description');
```

Comment:
Please add checks regarding contents of meta-tags. Including escaping of special characters and new-lines